src/platform-includes/performance/opentelemetry-setup/without-java-agent/java.spring-boot.mdx

```diff
@@ -0,0 +1,54 @@
+#### Initializing OpenTelemetry
+
+```Java
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+
+import io.sentry.opentelemetry.SentryPropagator;
+import io.sentry.opentelemetry.SentrySpanProcessor;
+
+
+SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+    .addSpanProcessor(new SentrySpanProcessor())
+    .build();
+
+OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+    .setTracerProvider(sdkTracerProvider)
+    .setPropagators(ContextPropagators.create(new SentryPropagator()))
+    .buildAndRegisterGlobal();
+```
+
+#### Initializing Sentry
+
+You have to set the `instrumenter` option to `otel`. This disables all Sentry instrumentation and relies on the chosen OpenTelemetry tracers for creating spans.
+
+```properties {filename:application.properties}
+sentry.dsn=___DSN___
+sentry.traces-sample-rate=1.0
+# enable this to see more logs
+sentry.debug=false
+# set the instrumenter to use OpenTelemetry instead of Sentry
+sentry.instrumenter=otel
+```
+
+You also have to manually provide the `OpenTelemetryLinkErrorEventProcessor` bean to ensure errors are properly linked to transactions that were created by the OpenTelemetry integration:
```

Comment:
Decided to keep this as is for now as there doesn't seem to be an easy way to detect whether the `sentry-opentelemetry-agent` is used or not and this seems like a rather niche use case.